tox.ini

```diff
@@ -4,4 +4,32 @@ envlist =
     py35,
     py36,
     py37,
+    check,
 isolated_build = true
+
+[testenv]
+description = run the test driver with {basepython}
+deps = -rrequirements-dev.txt
```

Comment:
-rr

Reply:
intentional. silly, but intentional. i have no clue how that's parsed.